typescript-react.md

```diff
@@ -0,0 +1,26 @@
+# Typescript/React
+
+_Spec v1 (2021-11-05)_
+
+## Options
+
+There's two alternatives from which you can choose from:
+
+1. Create a generic one-page scroller where the hero element has a background video served from IPFS.
+2. Using the [Vanilla SDK](https://www.npmjs.com/package/@vanilladefi/sdk), create a Next.js page with the list of users utilizing React 18 and server components.
+
+## Requirements
+
+1. Use Typescript
```

Comment:
And React 18? Should we specify the TS version too?

Reply:
Or hmm, did you mean that alternative 1) can be implemented in something else than React?

Reply:
React 18 because server components are only available in it. That said, we should specify React as a requirement for the first one as well.

Reply:
Maybe the first alternative is missing some oomph or interactivity? Should there be at least some state that's being handled?

Reply:
Good point and I agree, there should be at least some state management included in this challenge :thinking:  

What's the most purposeful way then, state changes by reacting to external data changes (polling graphql? etc) or by having user interaction? Including both may bloat the scope of the challenge too much.

Reply:
How about adding simple "Subscribe to newsletter" form. That should handle quite many use cases for state, handling controlled inputs, loading states, success and error? We could setup up mock api that randomly sends back either 200 OK or some server error?